src/Swarm/Game/WorldGen.hs

```diff
@@ -17,9 +16,10 @@ import Data.Enumeration
 import Data.Hash.Murmur
 import Data.Int (Int64)
 import Data.List (find)
-import Data.Maybe (fromMaybe)
+import Data.Maybe (fromMaybe, mapMaybe)
+import Data.Set qualified as S
 import Data.Text (Text)
-import qualified Data.Text as T
+import Data.Text qualified as T
```

Comment:
Hah, post-qualified import is nice :grinning: 

Maybe we could have it in default extensions (along with the rest of [GHC2021](https://github.com/ghc-proposals/ghc-proposals/blob/ghc2021/proposals/0000-ghc2021.rst#proposed-change)) and add [-Wprepositive-qualified-module](https://ghc.gitlab.haskell.org/ghc/doc/users_guide/using-warnings.html#ghc-flag--Wprepositive-qualified-module).

Reply:
This is not very related to this PR so I made #419, let us discuss it there. :slightly_smiling_face: